posts/mathlib-changelog.md

```diff
@@ -0,0 +1,55 @@
+---
+author: "David Chanin"
+category: "Community projects"
+date: 2022-07-28 09:35:23 UTC+02:00
+description: ""
+has_math: false
+link: ""
+slug: mathlib-changelog
+tags: ""
+title: Introducing Mathlib Changelog
+type: text
+---
+
+![mathlib-changelog sample page](/images/changelog_lemma.png)
+
+Tldr; check out [mathlib-changelog.org](https://mathlib-changelog.org) to explore the historical changes to mathlib, and find out what happened to that lemma you were using.
+
+<!-- TEASER_END -->
+
+## A changelog for mathlib
+
+As a beginner in the world of Lean and mathlib (and formal math in general), once I started branching out from the offical tutorials and the excellent [Natural Numbers Game](https://www.ma.imperial.ac.uk/~buzzard/xena/natural_number_game/) and started looking at proofs online I found that I kept getting confusing errors, even when typing verbatim what I would see on the screen. Searching on Google didn't help, and I often couldn’t find any reference to some of the definitions and lemmas I was seeing in the mathlib docs.
```

Comment:
```suggestion
As a beginner in the world of Lean and mathlib (and formal math in general), once I started branching out from the offical tutorials and the excellent [Natural Numbers Game](https://www.ma.imperial.ac.uk/~buzzard/xena/natural_number_game/) and started looking at proofs online, I found that I kept getting confusing errors, even when typing verbatim what I would see on the screen. Searching on Google didn't help, and I often couldn’t find any reference to some of the definitions and lemmas I was seeing in the mathlib docs.
```
I like the change, thanks! This comma fixes a mental parse error for me ("proofs online I found").

Reply:
👍 good call! Definitely easier to read that way